### ISSUE TITLE
Sequence names not in plmblast.py output

### ISSUE THREAD

**Issue by seanrjohnson** [labstructbioinf/pLM-BLAST#29]
When I run `plmblast.py` (with some patches to make it not crash, see #23), The output qid and sid are numbers. It would be great if instead of numbers, they were the actual names of the sequences as taken from the original fasta files.

See this screenshot of my output, if I want to know what my top hits are, I have to look through the database fasta file and figure out what the 0th, 3234th, 2172nd, etc sequences are. Which seems inconvenient.
![image](https://github.com/labstructbioinf/pLM-BLAST/assets/28655920/126868df-0dd2-4242-985b-934a23df4d16)

**Comment by Argusmocny** [comment]
This will we fixed in upcoming update (this week). Thanks for pointing this out.

**Comment by Argusmocny** [comment]
@seanrjohnson fixed

**Comment by papelypluma** [comment]
Hi @Argusmocny. I only realized that the qid and sid were numbers in the output I got after running the job for almost 2 weeks. I understand that there has been a fix/enhancement to this, but to come up with a quick solution on my end (without re-running the job) I'd like to ask whether the qid's and sid's numbers are 0-based indices following the order they appear in the fasta file I used to build the query and the reference database. 

Thanks in advance!
